### ISSUE TITLE
[Package Request] Prometheus

### ISSUE THREAD

**Issue by muety** [SynoCommunity/spksrc#5369]
### Software Name

Prometheus

### Brief Description

Prometheus is an open-source server monitoring solution

### Website

https://prometheus.io

### Documentation

https://prometheus.io/docs/introduction/overview/

### Build/Installation documentation

https://github.com/prometheus/prometheus#building-from-source

### Source code

https://github.com/prometheus/prometheus

### OpenSource License

Apache

**Comment by publicarray** [comment]
@muety do you mean the node-exporter  from Prometheus (to grab data from the NAS)? We already have a package for that: https://synocommunity.com/package/node-exporter.

**Comment by muety** [comment]
Agree, I also have a fear that Prometheus wouldn't run very well on the low-end NASs.

I'd want to use it (together with Grafana) for some home automation analytics and monitoring (power meters, etc.), where the data shall possibly only stay within my local network and where data volumes are not expected to grow large.

I'll give it a try to see how it performs and report back here then.
